services/src/main/java/org/fao/geonet/api/registries/vocabularies/KeywordsApi.java

```diff
@@ -631,7 +632,8 @@ private Object getKeyword(
         value = "/{thesaurus:.+}",
         method = RequestMethod.GET,
         produces = {
-            MediaType.TEXT_XML_VALUE
+            MediaType.TEXT_XML_VALUE,
+            MediaType.APPLICATION_XML_VALUE
```

Comment:
@fxprunayre 

I noticed that you added `MediaType.APPLICATION_XML_VALUE`
However this function uses the following content type.

`response.setContentType("text/xml");`

If someone calls the api requesting `MediaType.APPLICATION_XML_VALUE`, I think they would expect `MediaType.APPLICATION_XML_VALUE` and not `MediaType.TEXT_XML_VALUE`

Should 

`response.setContentType("text/xml");`

Be replaced with 
```
String acceptHeader = StringUtils.isBlank(request.getHeader(HttpHeaders.ACCEPT)) ? MediaType.TEXT_XML_VALUE : request.getHeader(HttpHeaders.ACCEPT);
response.setContentType(acceptHeader);
```